ddtrace/appsec/iast/_taint_tracking/_vendor/pybind11/include/pybind11/attr.h

```diff
@@ -10,8 +10,8 @@
 
 #pragma once
 
-#include "detail/common.h"
 #include "cast.h"
+#include "detail/common.h"
```

Comment:
Since this is a file from a vendored dependency, should we exclude it from our stylistic checks/fixups?

The BSD license _allows_ us to do whatever, so I'm fine with whatever.  Mostly curious about our thinking on this one.  This comment applies to other files in the patch, if it matters.